cmd/hooks/hooks.go

```diff
@@ -175,14 +177,16 @@ func PreRun(app *cli.App, db *postgres.Postgres) func(cmd *cobra.Command, args [
 		app.Rate = rateLimiter
 
 		// update config singleton with the instance id
-		configRepo := postgres.NewConfigRepo(app.DB)
-		instCfg, err := configRepo.LoadConfiguration(cmd.Context())
-		if err != nil {
-			log.WithError(err).Error("Failed to load configuration")
-		} else {
-			cfg.InstanceId = instCfg.UID
-			if err = config.Override(&cfg); err != nil {
-				return err
+		if cmd.Use != "bootstrap" && cmd.Use != "version" { // bootstrap & version don't need this
```

Comment:
Hmm, nice. I think `migrate` doesn't need this too. Can you add it?

Reply:
Done